.dockerignore

```diff
@@ -1,2 +1,2 @@
 *
-!ci/
+!docker/
```

Comment:
I could see if I can rearrange commits to make it a bit cleaner, but it was a very long-term work so at some point I just stopped caring. I could also squash everything in one commit I guess. I'll have a look and see what's the best I can make with the biggest outcome/effort ratio.